lib/src/rules/cascade_invocations.dart

```diff
@@ -51,6 +52,14 @@ someReference
 
 ''';
 
+Element _findElement(SimpleIdentifier identifier) {
+  if (identifier.bestElement is PropertyAccessorElement) {
```

Comment:
This is a hard one, it is linted but isSynthetic is not enough to discriminate. The bug is reproducible on master though, I wonder if it should be tracked on a different bug report.